src/modules/sharing/sharing.service.spec.ts

```diff
@@ -395,10 +400,305 @@ describe('Sharing Use Cases', () => {
+              encryptionKey: sharing.encryptionKey,
+              dateShared: sharing.createdAt,
+            }),
+          ]),
+          credentials: {
+            networkPass: user.userId,
+            networkUser: user.bridgeUser,
+          },
+          token: '',
+          role: 'OWNER',
+        }),
+      );
+    });
+
+    it('When no files are shared with the team, then it should return nothing', async () => {
+      jest
+        .spyOn(
+          sharingRepository,
+          'findFilesByOwnerAndSharedWithTeamInworkspace',
+        )
+        .mockResolvedValue([]);
+
+      const result = await sharingService.getSharedFilesInWorkspaces(
+        user,
+        teamId,
+        offset,
+        limit,
+        order,
+      );
+
+      expect(result).toEqual({
+        folders: [],
+        files: [],
+        credentials: {
+          networkPass: user.userId,
+          networkUser: user.bridgeUser,
+        },
+        token: '',
+        role: 'OWNER',
+      });
+    });
+
+    it('When there is an error fetching shared files, then it should throw', async () => {
+      const error = new Error('Database error');
+
+      jest
+        .spyOn(
+          sharingRepository,
+          'findFilesByOwnerAndSharedWithTeamInworkspace',
+        )
+        .mockRejectedValue(error);
+
+      await expect(
+        sharingService.getSharedFilesInWorkspaces(
+          user,
+          teamId,
+          offset,
+          limit,
+          order,
+        ),
+      ).rejects.toThrow(error);
+    });
+  });
+
+  describe('getSharedFoldersInWorkspace', () => {
+    const user = newUser();
+    const teamId = v4();
+    const offset = 0;
+    const limit = 10;
+    const order: [string, string][] = [['name', 'asc']];
+
+    it('When folders are shared with the team, then it should return the folders', async () => {
+      const sharing = newSharing();
+      const folder = newFolder();
+      folder.user = newUser();
+      sharing.folder = folder;
+      const foldersWithSharedInfo = [sharing];
+
+      jest
+        .spyOn(
+          sharingRepository,
+          'findFoldersByOwnerAndSharedWithTeamInworkspace',
+        )
+        .mockResolvedValue(foldersWithSharedInfo);
+
+      jest
+        .spyOn(folderUseCases, 'decryptFolderName')
+        .mockReturnValue({ plainName: 'DecryptedFolderName' });
+
+      jest.spyOn(usersUsecases, 'getAvatarUrl').mockResolvedValue('avatar-url');
+
+      const result = await sharingService.getSharedFoldersInWorkspace(
+        user,
+        teamId,
+        offset,
+        limit,
+        order,
+      );
+
+      expect(result).toEqual(
+        expect.objectContaining({
+          folders: expect.arrayContaining([
+            expect.objectContaining({
+              sharingId: sharing.id,
+              encryptionKey: sharing.encryptionKey,
+              dateShared: sharing.createdAt,
+              sharedWithMe: user.uuid !== sharing.folder.user.uuid,
```

Comment:
Set the value directly, not a logical comparison. Otherwise, this line would need to be tested also, which is what we want to avoid in a test.